10-std.tex

```diff
@@ -539,3 +539,145 @@ \section{Input/Output}
 
 \section{Sys/sys}
 \label{std-sys}
+
+\section{Remoting}
+\label{std-remoting}
+
+Haxe remoting is a way to talk between different platforms. With Haxe remoting we can communicate transparently, send data and call methods, between server and client side.
+
+\subsection{Remoting Connection}
+\label{std-remoting-connection}
+
+In order to use remoting, there must be a connection established first. There are two kinds of Haxe Remoting connections: 
+\begin{description}
+	\item[\expr{haxe.remoting.Connection}] is used for \emph{synchronous connections}, where the results can be directly obtained when calling a method. 
+	\item[\expr{haxe.remoting.AsyncConnection}] is used for \emph{asynchronous connections}, where the results are events that will happen later in the execution process.
+\end{description}
+
+\paragraph{Start a connection}
+Haxe remoting can be used to communicate across different platforms. There are some target-specific constructors with different purposes to set up a connection:
```

Comment:
First sentence repeats meaning of first sentence of the paragraph. Please review. Also review the second sentence. instead of "to set up a connection", consider "that can be used to set up a connection" to clarify meaning.